src/test/mir-opt/byte_slice.rs

```diff
@@ -0,0 +1,15 @@
+// compile-flags: -Z mir-opt-level=0
+
+fn main() {
+    let x = b"foo";
+    let y = [5u8, b'x'];
+}
+
+// END RUST SOURCE
+// START rustc.main.EraseRegions.after.mir
+// ...
+// _1 = const b"102111111";
```

Comment:
This... doesn't look right.

Reply:
Idk what kind of brainfart went on with me there.